_drafts/best-of-binutils.md

```diff
@@ -0,0 +1,1011 @@
+---
+title: "Best of GNU Binutils and other ELF introspection tools"
+description: "A guide of how to make the most of GNU binutils to examine binaries using tools such as
+readelf, nm, and objdump with practical examples"
+image: ""
+tags: [better-firmware, fw-code-size]
+author: chris
+---
+
+There's a lot that takes place between the C code you write
+and the binary that winds up executing on a device. Understanding how to look at and inspect
+what is emitted by the compiler saves time and improves can improve your efficiency in many areas
+of the development lifecycle -- such
+as debugging system problems, identifying issues with compilers or
+debug info emitted, reducing the size of binaries, and optimizing an application for performance and minimal latency.
+
+<!-- excerpt start -->
+
+In this article, we will explore some of the best binary introspection tools available such as GNU
```

Comment:
Would love for you to make this more personal and less click-baity, likely by saying "we will explore some of my favorite binary introspection tools, such as GNU Binutils."